docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -581,3 +600,84 @@ If processing of any rule fails, a partial error outputs the ID and/or name of t
 --------------------------------------------------
 
 <1> dev:[] `execution_summary` is under development and its schema may change.
+
+*Example 3, Dry run*
+
+The following request will validate that the `add_index_patterns` bulk action can be successfully applied to three rules. Each rule (which specified by its rule ID) is different: one is a prebuilt Elastic rule, another is a custom machine learning rule, and another is a custom query rule. Because dry run mode is enabled, changes to these rules will not be permanently applied and saved to {es}.
+
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/rules/_bulk_action?dry_run=true
+{
+    "action": "edit",
+    "edit": [
+        {
+            "value": [
+                "test-*"
+            ],
+            "type": "add_index_patterns"
+        }
+    ],
+    "ids": ["81aa0480-06af-11ed-94fb-dd1a0597d8d2", "dc015d10-0831-11ed-ac8b-05a222bd8d4a", "de8f5af0-0831-11ed-ac8b-05a222bd8d4a"]
+}
+--------------------------------------------------
+
+[discrete]
+===== Response code
+
+`500`::
+    Indicates partial bulk action failure.
+
+[discrete]
+===== Response payload
+
+The `attributes.errors` section of the response shows that two rules failed to update and one succeeded. The same results would be returned if you ran the same request without dry run mode enabled.
+Notice that there are no arrays in `attributes.results`. In the dry run mode, rule updates are not applied and saved to {es}, which means the endpoint won't return results for rules that have been `updated`, `created`, or `deleted`.
```

Comment:
```suggestion
Notice that there are no arrays in `attributes.results`. In the dry run mode, rule updates are not applied and saved to {es}, so the endpoint wouldn't return results for rules that have been `updated`, `created`, or `deleted`.
```

Reply:
I actually like "won't" better than "wouldn't" here. `Are not... do not` seems more consistent to me than `are not... would not`

Reply:
I agree! I'm also not a fan of using future tense verbs but couldn't think of an acceptable alternative. Thanks, @benironside !